udata_front/settings.py

```diff
@@ -33,3 +33,16 @@
 GUIDES_ORGANIZATION_URL = 'https://guides.etalab.gouv.fr/data.gouv.fr/creer-rejoindre-organisation/'
 GUIDES_DATASET_URL = 'https://guides.etalab.gouv.fr/data.gouv.fr/publier-jeu-de-donnees/'
 GUIDES_REUSE_URL = 'https://guides.etalab.gouv.fr/reutilisation/publier-reutilisation/'
+
+# CAPTCHEtat
+
+# You can activate CaptchEtat, a captcha.com integration by providing
+# CAPTCHETAT_BASE_URL, CAPTCHETAT_OAUTH_BASE_URL, CAPTCHETAT_CLIENT_ID and CAPTCHETAT_CLIENT_SECRET
+CAPTCHETAT_BASE_URL = None
+CAPTCHETAT_OAUTH_BASE_URL = None
+CAPTCHETAT_CLIENT_ID = None
+CAPTCHETAT_CLIENT_SECRET = None
+CAPTCHETAT_OAUTH_TOKEN_URL = f'{CAPTCHETAT_OAUTH_BASE_URL}/api/oauth/token'
```

Comment:
Sadly `CAPTCHETAT_OAUTH_TOKEN_URL` uses `CAPTCHETAT_OAUTH_BASE_URL` defined above, not the one loaded in the current_app (and possibly overridden). We could concatenate both in `api.py` instead of formatting it here? Or defining the pattern there?